src/ElectronApp.ts

```diff
@@ -48,12 +54,36 @@ export default class ElectronApp {
    * Some APIs can only be used after this event occurs.
    */
   private handleAppReady = () => {
+    this.setDeepLink(process.argv.pop())
     this.createSystemTray()
     this.createMainWindow()
     this.handleOpenAtLogin(preferences.data || {})
+    this.openWindow()
     EventBus.emit(EVENTS.ready, this.tray)
   }
 
+  private handleSecondInstance = (_: electron.Event, argv: string[]) => {
+    // Windows deep link support
+    Logger.info('SECOND INSTANCE ARGS', { argv })
+    this.setDeepLink(argv.pop())
```

Comment:
Setting deep link to a var proved much more stable, then clearing it once window is opened.